### ISSUE TITLE
Missing Variables, Constants, and Flags

### ISSUE THREAD

**Issue by TehFishey** [TehFishey/stellaris-crisis-managers#3]
A few scripted variables, constants, and flags appear to be missing from the crisis manager files: 

Variables:
CmtVarLgateCycleStandard
CmtVarLgateCycleStandard

Constants:
@CmtT101_ConstLgateClusterType
@CmtT101_ConstCrisisPower_Endgame

Flags:
CmtFlagUprisingEdition
CmtFlagFeeCompatPatch
CmtFlagConfigViewerShowing1025

Determine the intended purposes of these objects; add them with proper values if they are necessary, or remove them if not.

**Comment by TehFishey** [comment]
CmtVarLgateCycleStandard is deprecated since FirePrince's changes; replace with CmtVarLgateCycleCommon.

@CmtT101_ConstLgateClusterType appears to be depreciated constants which are included for backwards compatability purposes. It can probably be safely removed (along with other backwards compat stuff), considering that the mod is now for a new version of the game (savegame compat with previous editions is impossible)

@CmtT000_ConstCrisisPower_Endgame seems to be an obsolete constant; the same template variable is properly set using @CmtT000_ConstCrisisStrengthEnd at other points in the script. Consider removing.

The first two flags are to recognize other editions of the game which no longer exist in this version. Leaving them as they are shouldn't hurt until we can properly implement the features from those mods (and decide for 100% that we won't just update them).

CmtFlagConfigViewerShowing1025 is a display flag for a config viewer that no longer exists. It's probably fine to leave in right now.

**Comment by F1r3Pr1nc3** [comment]
Yes, it should all be easy doable. I made a beginning (Constants done)

**Comment by F1r3Pr1nc3** [comment]
scripted effect `CmtEffectCreateFleet_KhanConstructor` is doubled in the Mid-Game edition. In one file header says End-Game the other Mid-Game... File: z_Cmt333b_CrisisAuxiliaryShip_scripted_effects.txt vs  z_Cmt332b_CrisisAuxiloaryFleet_scripted_effects.txt
